_layouts/coe.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+---
+
+<section class="usa-grid usa-section">
+    <h1>{{ page.title }}</h1>
+    <div class="usa-width-two-thirds">
+        <p class="intro">{{ page.tagline }}</p>
+
+        <div class="page-nav">
+            <a href="#service offerings">Service Offerings</a> | {% if page.service-catalog %}<a href="#service catalog">Service Catalog</a> | {% endif %}<a href="#coe-updates">Latest Updates</a> | {% for item in page.page-nav.children %}<a href="{{ site.baseurl }}/work/{{ item.url-fragment }}">Our Work at {{ item.agency }}</a> {% if forloop.last == false %} | {% endif %}{% endfor %}
+        </div>
+
+      <p>{{ page.intro }}</p>
+
+      <h2 id="service-offerings">Service Offerings</h2>
+      <ul>
+      {% for offering in page.service-offerings %}
+        <li>{{ offering }}</li>
+      {% endfor %}
+      </ul>
+
+      {% if page.service-catalog %}
+        <h2 id="service-catalog">Service Catalog</h2>
+        {% for child in page.service-catalog.children %}
+          {% if child.description %}
+            <p>{{ child.description }}</p>
+          {% endif %}
+          {% if child.list %}
+            <ul>
+              {% for item in child.list %}
+                <li>{{ item }}</li>
+              {% endfor %}
+            </ul>
+          {% endif %}
+        {% endfor %}
+      {% endif %}
+
+      <h2 id="coe-updates">Latest Updates</h2>
+      {% assign filtered_posts = site.posts | where: 'initiative', page.title %}
+      {% for post in filtered_posts %}
+          <div class="article-head">
+              <em class="date">{{ post.date | date: '%B %d, %Y' }}</em>
+              <h3 class="title"><a href="{{site.baseurl}}{{post.url}}">{{post.title}}</a></h3>
+              {{ post.content }}
+          </div>
+      {% endfor %}
```

Comment:
This is the same loop that exists in #126, just consolidated into a single layout.